frontend/src/concepts/pipelines/content/tables/pipelineRun/CustomMetricsColumnsModal.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import {
+  Button,
+  Checkbox,
+  DataListDragButton,
+  DragDrop,
+  Draggable,
+  DraggableItemPosition,
+  Droppable,
+  Flex,
+  FlexItem,
+  Label,
+  Modal,
+  ModalBoxBody,
+  ModalVariant,
+  Stack,
+  StackItem,
+  Tooltip,
+} from '@patternfly/react-core';
+import { getMetricsColumnsLocalStorageKey } from './utils';
+
+interface MetricsColumn {
+  id: string;
+  checked: boolean;
+}
+
+interface CustomMetricsColumnsModalProps {
+  columns: MetricsColumn[];
+  experimentId: string | undefined;
+  onClose: () => void;
+}
+
+export const CustomMetricsColumnsModal: React.FC<CustomMetricsColumnsModalProps> = ({
+  onClose,
+  experimentId,
+  ...props
+}) => {
+  const [columns, setColumns] = React.useState(props.columns);
+  const [dragDropText, setDragDropText] = React.useState('');
```

Comment:
is this just for debugging im guessing?

Reply:
Yes, I didn't need this, but looking closer at PF's documentation, I noticed there is a newer version of the drag/drop component, so I'll upgrade to that while I'm at it.